source/game.c

```diff
@@ -483,6 +484,33 @@ static const Rect GAME_OVER_ANIM_RECT       = {11,      8,       23,     28};
 static const BG_POINT NEW_RUN_BTN_DEST_POS  = {15,      26};
 static const Rect NEW_RUN_BTN_SRC_RECT      = {0,       30,      4,      31};
 
+// Flaming score animation frames
+#define NB_SCORE_FLAMES_FRAMES 8
```

Comment:
```suggestion
#define NUM_SCORE_FLAMES_FRAMES 8
```

Just a more consistent name to be clearer.